inference_experimental/inference_exp/models/auto_loaders/ranking.py

```diff
@@ -49,8 +50,10 @@ def rank_model_packages(
     # discarded in the previous stage.
```

Comment:
re-reading the whole comment her made me think maybe at some point might consider giving a mechanism to configure ranking preference.  Maybe configuring explicitly on AutoModel by passing constraints is already enough (but only gives control over filtering, not ranking).  Just a thought not something we need to worry about now.